docs/DeveloperGuide.md

```diff
@@ -504,8 +562,8 @@ Figure ???: <i>The sequence diagram of the execution of StatsRecipeTopCommand </
 7. `StatsRecipeTopCommand` calls `getMostMadeRecipeList` of `Model` and a list of String pairs is returned. This list is encapsulated in `CommandResult` which is then returned to `Logic`.
 
 
-<h2 style="background-color: #1077ff">TODO: stuff below isn't done</h2>
-
+![UndoRedoState0](images/RecipetopCmdStatsBox.png)
```

Comment:
can you move the image into `images/dg/`

Reply:
Oh, okie